requirements.txt

```diff
@@ -1,2 +1,3 @@
 piexif==1.1.3
 win32-setctime==1.0.3
+tqdm==4.56.0
```

Comment:
I think here is missing the requirement of loguru isn't it?

```suggestion
tqdm==4.56.0
loguru==0.5.3
```

Reply:
Loguru is not part of any pull request so i haven't finished including it.